vcx/libvcx/src/connection.rs

```diff
@@ -477,8 +477,19 @@ pub fn build_connection_with_invite(source_id: &str, details: &str) -> Result<u3
     let details:Value = serde_json::from_str(&details)
         .or(Err(ConnectionError::CommonError(error::INVALID_JSON.code_num)))?;
 
-    let invite_details:InviteDetail = serde_json::from_value(details)
-        .or(Err(ConnectionError::CommonError(error::INVALID_INVITE_DETAILS.code_num)))?;
+    let invite_details:InviteDetail = match serde_json::from_value(details.clone()) {
```

Comment:
Hope it doesn't change the JSON keys that are returned in serialize connection api call. If it does, then our functionality would break

Reply:
It shouldn't.  I didn't change any structs.  I just changed it so that if it fails to deserialize it converts it to abbreviated json and then tries to deserlize again.

Reply:
ok, sounds good then